### DISCUSSION TITLE
can i fork packager?

### DISCUSSION THREAD

**Discussion by ghost** [TurboWarp/packager#497] [Q&A]
i want to fork packager is that okay?

**Comment by GarboMuffin** [comment]
The packager is fully open source. You can do *basically anything* you want with the packager and its source code without asking for permission.

https://github.com/TurboWarp/packager/blob/master/LICENSE

**Comment by ghost** [comment]
ty but when i fork it i used github pages but the index.html file is missing is in a folder?
flosshost won't work because is do things limiting people useing it

**Reply by GarboMuffin** [reply]
The packager is not a simple website where you can just edit index.html. There is a mandatory build step that requires the command line. Deploying to GitHub pages uses a manually-run GitHub Actions script.

Please read https://github.com/TurboWarp/packager/blob/master/README.md. I spent time writing this for a reason.

**Comment by ghost** [comment]
im on a chromebook sorry

**Reply by GarboMuffin** [reply]
You'll have to find a service such as repl.it that lets you access a real programming environment.

**Comment by ghost** [comment]
wait you mean i need vscode

**Reply by GarboMuffin** [reply]
You need a service that lets you access a full command line and development environment. Doesn't matter what you use.

**Comment by ghost** [comment]
what do you use do to it

**Reply by GarboMuffin** [reply]
The software I use will not run easily (or at all) on a Chromebook, especially if it's one provided by your school

I linked to replit above. use that. it should work

**Comment by ghost** [comment]
can you tell me the name

**Reply by GarboMuffin** [reply]
Visual Studio Code
